pkg/clusteragent/orchestrator/status.go

```diff
@@ -69,6 +67,25 @@ func GetStatus(ctx context.Context, apiCl kubernetes.Interface) map[string]inter
 		}
 	}
 	status["OrchestratorEndpoints"] = endpoints
+	setCacheInformationDCAMode(status)
+	setCollectionIsWorkingDCAMode(status)
+
+	// rewriting DCA Mode in case we are running in cluster check mode.
+	if config.Datadog.GetBool("cluster_checks.enabled") {
+		status["CLCEnabled"] = true
+		status["CacheNumber"] = "No Elements in the cache, since collection is run on CLC Runners"
+		status["CollectionWorking"] = "The collection is not running on the DCA but on the CLC Runners"
+	}
```

Comment:
I don't think this condition `config.Datadog.GetBool("cluster_checks.enabled")` is enough to achieve what you wanted.
1. `cluster_checks.enabled` just enables dispatching cluster check configurations, and has nothing to do with cluster check runners, TBH I don't see how the DCA can know whether the node agents or the cluster check runners are consuming the cluster checks (with the current utils and configs we have)
2. Even if you know that cluster check runners are enabled, it doesn't necessarily mean that they run the orchestrator checks. It depends on whether the orchestrator checks have the bool field `cluster_check` enabled or not

Reply:
Thanks for the check. 
I added this here: 9e4a40b10aa792c3b42de02426bb03be7a27005f let me know what you think. Furthermore I updated the PR description with output of each cases.